scripts/path.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -e
+
+# Delete the "patches" directory is
+[ -d "./patches" ] && rm -r patches
```

Comment:
I'd suggest not deleting the whole dir but rather its contents. If you're worried about the first run when the directory is empty or may not exist yet, you could `mkdir` it.

```sh
mkdir -p patches
rm -rf patches/*
```

Reply:
Yes much better.